src/tests/payment/test_stripe_webhook.py

```diff
@@ -0,0 +1,371 @@
+import hashlib
+import hmac
+import json
+import time
+from datetime import datetime, timedelta
+from decimal import Decimal
+from unittest import mock
+
+import pytest
+from django.test import Client
+from django.urls import reverse_lazy
+from django.utils import timezone
+
+from payment.models import CustomerDetail, Subscription, User
+
+PASSWORD = "password1"
+FIRSTNAME = "John"
+LASTNAME = "Doe"
+EMAIL = "johndoe@example.fr"
+USERNAME = EMAIL
+EMAIL_HOST_USER = "support@ordopro.fr"
+
+STRIPE_WEBHOOK_SECRET = "whsec_testsecret"
+
+
+@pytest.fixture
+def user(db):
+    """Creates and yields a new user."""
+    user = User.objects.create(
+        username=USERNAME, first_name=FIRSTNAME, last_name=LASTNAME, email=EMAIL
+    )
+    user.set_password(PASSWORD)
+    user.save()
+    yield user
+    user.delete()
+
+
+@pytest.fixture
+def client():
+    return Client()
+
+
+@pytest.fixture
+def payload():
+    """Fixture to generate a test payload."""
```

Comment:
That comment is so generic that it becomes useless.
Maybe:
> Stripe session checkout completed payload

Or something like that, GPT could probably come up with something better.
Or remove the comment but rename the fixture to be:
```
def checkout_session_completed_payload():
```

Reply:
I’ve refactored the fixture name to checkout_session_completed_payload() to make it more descriptive, as suggested, and removed the generic comment.